express-api/src/typeorm/Entities/Users_Roles_Claims.ts

```diff
@@ -0,0 +1,201 @@
+import { UUID } from 'crypto';
+import {
+  Entity,
+  Column,
+  CreateDateColumn,
+  ManyToOne,
+  Index,
+  JoinColumn,
+  PrimaryColumn,
+  OneToMany,
+  Relation,
+} from 'typeorm';
+import { Agencies } from './Agencies';
+
+@Entity()
+export class Users {
+  @PrimaryColumn({ type: 'uuid' })
+  Id: UUID;
+
+  @ManyToOne(() => Users, (User) => User.Id)
+  @JoinColumn({ name: 'CreatedById' })
+  CreatedById: Users;
+
+  @CreateDateColumn()
+  CreatedOn: Date;
+
+  @ManyToOne(() => Users, (User) => User.Id, { nullable: true })
+  @JoinColumn({ name: 'UpdatedById' })
+  UpdatedById: Users;
+
+  @Column({ type: 'timestamp', nullable: true })
+  UpdatedOn: Date;
+
+  @Column({ type: 'character varying', length: 25 })
+  @Index({ unique: true })
+  Username: string;
+
+  @Column({ type: 'character varying', length: 100 })
+  DisplayName: string;
+
+  @Column({ type: 'character varying', length: 100 })
+  FirstName: string;
+
+  @Column({ type: 'character varying', length: 100, nullable: true })
+  MiddleName: string;
+
+  @Column({ type: 'character varying', length: 100 })
+  LastName: string;
+
+  @Column({ type: 'character varying', length: 100 })
+  @Index({ unique: true })
+  Email: string;
+
+  @Column({ type: 'character varying', length: 100, nullable: true })
+  Position: string;
+
+  @Column('bit')
+  IsDisabled: boolean;
+
+  @Column('bit')
+  EmailVerified: boolean;
+
+  @Column('bit')
+  IsSystem: boolean;
+
+  @Column({ type: 'character varying', length: 1000, nullable: true })
+  Note: string;
+
+  @Column({ type: 'timestamp', nullable: true })
+  LastLogin: Date;
+
+  @ManyToOne(() => Users, (User) => User.Id, { nullable: true })
+  @JoinColumn({ name: 'ApprovedById' })
+  ApprovedById: Users;
+
+  @Column({ type: 'timestamp', nullable: true })
+  ApprovedOn: Date;
+
+  @Column({ type: 'uuid', nullable: true })
+  @Index({ unique: true })
+  KeycloakUserId: string;
+
+  @Column({ name: 'AgencyId', type: 'varchar', length: 6 })
```

Comment:
Can we make the AgencyId column nullable to accommodate the users in PIMS that have no agencies?

`@Column({ name: 'AgencyId', type: 'varchar', length: 6, nullable: true })`